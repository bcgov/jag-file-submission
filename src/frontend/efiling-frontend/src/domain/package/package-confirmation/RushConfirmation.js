/* eslint-disable no-unused-vars */
import React from "react";
import PropTypes from "prop-types";
import { ConfirmationPopup } from "shared-components";

export default function RushConfirmation({ setShow }) {
  const modal = {};
  const mainButton = <></>;
  const confirmButton = <></>;
  const cancelButton = <></>;

  const handleClose = () => setShow(false);
  const handleShow = () => setShow(true);
  const handleConfirm = () => {};

  const confirmationPopup = {
    modal,
    mainButton: { ...mainButton, onClick: handleShow },
    confirmButton: { ...confirmButton, onClick: handleConfirm },
    cancelButton: { ...cancelButton, onClick: handleClose },
  };

  return (
    <ConfirmationPopup
      modal={modal}
      mainButton={mainButton}
      confirmButton={confirmButton}
      cancelButton={cancelButton}
    />
  );
}

RushConfirmation.propTypes = {
  setShow: PropTypes.func.isRequired,
};
