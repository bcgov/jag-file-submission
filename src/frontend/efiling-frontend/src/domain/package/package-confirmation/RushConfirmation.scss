.rush-modal {
  border-radius: 30px;

  .modal-header{
    margin: 20px 20px 0 20px; 
  }

  .modal-body {
    margin: 0 20px 20px 20px;
  }
  
  p, li {
    font-size: 18px;
  }
}