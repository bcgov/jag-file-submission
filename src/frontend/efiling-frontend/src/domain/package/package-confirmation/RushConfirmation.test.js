import React from "react";
import { render } from "@testing-library/react";

import RushConfirmation from "./RushConfirmation";

describe("RushConfirmation Component", () => {
  const setShow = jest.fn();

  test("Matches the snapshot", () => {
    const { asFragment } = render(<RushConfirmation show setShow={setShow} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
