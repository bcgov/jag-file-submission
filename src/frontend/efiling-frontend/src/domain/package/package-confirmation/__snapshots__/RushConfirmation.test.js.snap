// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RushConfirmation Component Matches the snapshot 1`] = `<DocumentFragment />`;
